src/commands/raid-leaders/StartAfkCheck.ts

```diff
@@ -121,8 +122,17 @@ export class StartAfkCheck extends BaseCommand {
             });
             return 0;
         }
+        
+        // Step 5: Check if there's a current headcount running. This is structured here for easier selections by the RL.
+        
+        const activeHeadcount = HeadcountInstance.ActiveHeadcounts.find(headcount => headcount.getHeadcountInfoObject()!.memberInit == ctx.interaction.user.id)
 
-        // Step 5: Start it
+        if (activeHeadcount) {
+            await activeHeadcount.convertHeadcount(ctx.member!, vcToUse);
+            return 0;
```

Comment:
See previous comment.  Would change this to abort and not return 0.  Also consider the case that there are multiple active headcounts up, would abort ALL of them.